Sources/swift-format/main.swift

```diff
@@ -69,18 +69,36 @@ fileprivate func main(_ arguments: [String]) -> Int32 {
 ///   - transform: A closure that performs a transformation on a specific source file.
 private func processSources(
   from paths: [String], configurationPath: String?,
-  transform: (FileHandle, String, Configuration) -> Int
+  transform: @escaping (FileHandle, String, Configuration) -> Int
 ) -> Int32 {
   var result = 0
+  let concurrentQueue = DispatchQueue(label: "com.queue.Concurrent", qos: .userInitiated, attributes: .concurrent)
+  let group = DispatchGroup()
+  let lock = NSLock()
   for path in FileIterator(paths: paths) {
-    guard let sourceFile = FileHandle(forReadingAtPath: path) else {
-      stderrStream.write("Unable to create a file handle for source from \(path).\n")
-      stderrStream.flush()
-      return 1
```

Comment:
Previously this aborted the whole set of files. Now it seems like it would have "keep-going" behavior.

Reply:
+1, given that I assume this error should be an edge case hopefully this behavior change is ok